### PR TITLE
Don't re-export `Network.Wai.Parse.ParseRequestBodyOptions`

### PR DESCRIPTION
**Pull request by pbrinkmeier**
Does what it says, addresses #387 

Sorry for spamming you with PR's, got some time on my hand and am a fan of the project so I'd like to contribute.

### CODE REVIEW

**Comment by pbrinkmeier** [comment]
Fixed the conflict in `changelog.md`

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## next [????.??.??]
+
 * Fixed cookie example from `Cookie` module documentation. `getCookie` Function would return strict variant of `Text`. Will convert it into lazy variant using `fromStrict`. 
+* Remove re-export of `Network.Wai.Parse.ParseRequestBodyOptions` from `Web.Scotty` and `Web.Scotty.Trans`. This is a type from `wai-extra` so exporting it is unnecessary.
```
Comment (ocramz) [inline changelog.md:4]: Could you list it under the Breaking Changes for "next"? Thanks!

**Review by ocramz — CHANGES_REQUESTED** [review]
Thank you, 👍 ! Just a minor thing with the changelog entry.

File: changelog.md (hunk 1 above)
Comment (pbrinkmeier) [reply changelog.md]: Moved it.

**Review by ocramz — APPROVED** [review]
👍 Thank you!

**Pull request merged**